pkg/virt-handler/vm_test.go

```diff
@@ -803,15 +797,15 @@ var _ = Describe("VirtualMachineInstance", func() {
 			controller.Execute()
 		})
 
-		It("should move VirtualMachineInstance to Failed if configuring the networks on the virt-launcher fails", func() {
+		It("should move VirtualMachineInstance to Failed if configuring the networks on the virt-launcher fails with critical error", func() {
 			vmi := v1.NewMinimalVMI("testvmi")
+			vmi.UID = vmiTestUUID
 			vmi.ObjectMeta.ResourceVersion = "1"
 			vmi.Status.Phase = v1.Scheduled
 
 			mockWatchdog.CreateFile(vmi)
 			vmiFeeder.Add(vmi)
-
-			network.SetupPodNetworkPhase1 = func(vm *v1.VirtualMachineInstance, pid int) error { return fmt.Errorf("SetupPodNetworkPhase1 error") }
+			mockIsolationResult.EXPECT().DoNetNS(gomock.Any()).Return(&network.CriticalNetworkError{Msg: "Critical SetupPodNetworkPhase1 error"}).AnyTimes()
```

Comment:
Can this be more specific to `Times(n)`?

Reply:
Similar for all other occurrences

Reply:
done